source/_includes/puppet-collections/_pkgutil_command_output.md

```diff
@@ -0,0 +1,11 @@
+    Package "puppet-agent-<AGENT-VERSION>-1-installer.pkg":
+       Status: signed by a certificate trusted by Mac OS X
+       Certificate Chain:
+        1. Developer ID Installer: PUPPET LABS, INC. (VKGLGN2B6Y)
```

Comment:
If the `pkgutil` output will change with the new key, I need either updated output or instructions on how to generate updated output.